sources/packages/backend/libs/services/src/students-assessments/assessment-sequential-processing.service.ts

```diff
@@ -297,6 +259,104 @@ export class AssessmentSequentialProcessingService {
     ];
   }
 
+  /**
+   * Gets the full time program year contribution totals for the provided application numbers.
+   * @param applicationNumbers application numbers.
+   * @returns full time program year contribution totals.
+   */
+  async getFTProgramYearContributionTotals(
+    applicationNumbers: string[],
+  ): Promise<FTProgramYearContributionTotal[]> {
+    const totals = await this.applicationRepo
+      .createQueryBuilder("application")
+      .select(
+        "COALESCE(SUM(CAST(currentAssessment.workflowData -> 'calculatedData' ->> 'totalFederalFSC' AS NUMERIC)),0)",
```

Comment:
Was it not possible to use a more regular `SUM` without `CAST` and `COALESCE`?

Reply:
Changed it to have simple SUM and casting it to Numeric, Removed COALESCE by initializing with 0 in the result transformation phase.